packages/slider/README.md

```diff
@@ -1,64 +1,124 @@
+## API
 
-      ```<mwc-slider></mwc-slider>```
 
-  1. Install the Polymer CLI:
+### Properties/Attributes
 
-      ```npm i -g polymer-cli```
+| Name      | Type      | Default | Description
+| --------- | --------- |-------- | -----------
+| `value`   | `number`  | `0`     | Current value of the slider.
+| `min`     | `number`  | `0`     | Minimum value of the slider.
+| `max`     | `number`  | `100`   | Maximum value of the slider.
+| `step`    | `number`  | `0`     | Sets the step of the slider making it discrete. Set to 0 to make continuous. Note: if `step` is greater than 0 but less than 1, it will be rounded up to 1.
+| `pin`     | `bookean` | `false` | Shows the thumb pin on a discrete slider.
+| `markers` | `bookean` | `false` | Shows the marker on the slider's track of a discrete slider.
 
-  1. Run the development server and open a browser pointing to its URL:
+### Methods
 
-      ```polymer serve```
+| Name     | Description
+| -------- | -------------
+| `layout() => void` | Updates the internal size model. May be required if the slider is resized or repositioned horizontally.
 
-  > mwc-slider is published on [npm](https://www.npmjs.com/package/@material/mwc-slider) using JavaScript Modules.
-  This means it can take advantage of the standard native JavaScript module loader available in all current major browsers.
-  >
-  > However, since mwc-slider uses npm convention to reference dependencies by name, a light transform to rewrite specifiers to URLs is required to get it to run in the browser. The polymer-cli's development server `polymer serve` automatically handles this transform.
+### Listeners
+| Event Name    | Target   | Description
+| ------------- | -------- | -----------
+| `resize`      | `window` | Performs slider layout (passive).
+| `mousemove`   | `body`   | Move the slider thumb and set value after initial interaction (passive).
+| `mouseup`     | `body`   | Move the slider thumb and set value after initial interaction (passive).
+| `pointermove` | `body`   | Move the slider thumb and set value after initial interaction (passive).
+| `pointerup`   | `body`   | Move the slider thumb and set value after initial interaction (passive).
+| `touchmove`   | `body`   | Move the slider thumb and set value after initial interaction (passive).
+| `touchend`    | `body`   | Move the slider thumb and set value after initial interaction (passive).
 
-  Tools like [WebPack](https://webpack.js.org/) and [Rollup](https://rollupjs.org/) can also be used to serve and/or bundle mwc-slider.
+### Events
 
-## Supported Browsers
-
-The last 2 versions of all modern browsers are supported, including
-Chrome, Safari, Opera, Firefox, Edge. In addition, Internet Explorer 11 is also supported.
\ No newline at end of file
+| Event Name | Target       | Detail             | Description
+| ---------- | ------------ | ------------------ | -----------
+| `input`    | `mwc-slider` | `Slider`           | Fired when the slider is beginning to open.
```

Comment:
What does it mean for a slider to "open"?

Reply:
OMG I never updated the descriptions for this XD